Nfield.Quota/PreOrderQuotaFrameTraverser.cs

```diff
@@ -8,7 +8,7 @@ public class PreOrderQuotaFrameTraverser
     {
         public QuotaFrame Traverse(
             QuotaFrame frame,
-            Action<QuotaFrameLevel> levelOperation)
+            Action<QuotaFrameVariable, QuotaFrameLevel> levelOperation)
```

Comment:
Is there a particular reason why you chose to change the api and break backwards compatibility, rather than adding an overload and keep it?
I see that the version is correctly increased, but I wonder: will we have to maintain version 2 of the package as well after this gets released?

Reply:
I doubt if anyone but us uses this class, it is not documented.
So I do not have a big problem with changing undocumented classes, when someone uses this they have to change their code.